app/src/components/completed/TransactionDialog.tsx

```diff
@@ -27,6 +26,7 @@ import { TokenLogo } from '../TokenLogo'
 
 export const TransactionDialog = ({ tx }: { tx: CompletedTransfer }) => {
   const transferSucceeded = tx.result === TxStatus.Succeeded
+  const transferUndefined = tx.result === TxStatus.Undefined
```

Comment:
not the way to go. these are mutually exclusive so there's no reason to have two booleans to represent one single thing.

At worse, we could change this to the following if we want a better naming:

```suggestion
  const txStatus = tx.result
```

But deleting these `const`s and just using `tx.result` directly is the way to go imo.